hazelcast-sql/src/test/java/com/hazelcast/sql/misc/SqlNoDeserializationTest.java

```diff
@@ -127,24 +111,22 @@ private static PortableFactory portableFactory() {
 
     @Test
     public void testMember() {
-        try (SqlResult res = member.getSql().execute(SQL)) {
+        try (SqlResult res = instance().getSql().execute(SQL)) {
             for (SqlRow row : res) {
                 SqlRowImpl row0 = (SqlRowImpl) row;
 
                 row0.getObjectRaw(0);
                 row0.getObjectRaw(1);
-
-                checkFailure(row, true);
```

Comment:
I think that after changes it verifies something completely different than originally (`checkFailure()`s were removed, query now fails at execution).

Reply:
It verifies the equal behavior with exception that it fails-fast in `execute()` and return exception with `ERROR_KEY` (de)serialization.

Reply:
Previously it was verifying a failure on `row.getObject(index)` (for both key & value) - it wasn't failing on `execute()`. So the behaviour has changed, is it expected?